tools/Azure.Mcp.Tools.Foundry/src/Commands/ModelDeploymentCommand.cs

```diff
@@ -19,11 +19,9 @@ public sealed class ModelDeploymentCommand : SubscriptionCommand<ModelDeployment
     public override string Name => "deploy";
 
     public override string Description =>
-        """
-        Deploy a model to Azure AI Foundry.
-
-        This function is used to deploy a model on Azure AI Services, allowing users to integrate the model into their applications and utilize its capabilities. This command should not be used for Azure OpenAI Services to deploy OpenAI models.
-        """;
+    "Deploys (create) a model instance (e.g. GPT4o / gpt-4o, OpenAI, OSS, proprietary) in Azure AI Foundry as a named inference deployment in Azure AI Foundry, which's bound to a specified Azure AI Services resource, resource group, and subscription. " +
+    "Provide a deployment name plus model name and format to provision a persistent runtime endpoint addressable by that name, which enables applications to call models through a stable alias. " +
+    "Not for Azure OpenAI managed deployments—use the OpenAI deployment tool instead.";
```

Comment:
Can we unify how we tell agents to use tools for other tasks? We're using different types of phrasing for all 3 tools in this PR.